src/QuadForm/Lattices.jl

```diff
@@ -1569,3 +1588,37 @@ and which is maximal in `V` satisfying this property.
 """
 maximal_integral_lattice(::AbsSpace)
 
+################################################################################
+#
+#  Saturation
+#
+################################################################################
+
+@doc Markdown.doc"""
+    saturate(L::AbsLat) -> AbsLat
+
+Given a lattice `L` over a field `E`, return the closure $L\otimes E \cap O_E^n$
+of `L` in its rational span (see [`rational_span`](@ref)). Here $O_E$ is the base
+ring of `L` (see [`base_ring`](@ref)) and `n` is the degree of `L`.
+"""
```

Comment:
I think saturation is more of a computational tool for (pseudo)-matrices. 
For lattices we called it `primitive_closure` and it takes two lattices as arguments. 
So you will need `_primitive_closure_via_restriction_of_scalars` or something similar.

Reply:
The upshot is: do not implement `saturation` but rather `primitive_closure`. 
But we can allow the alias `saturate(L::AbsLat, M::AbsLat) = primitive_closure(L, M)`